x-pack/plugins/security_solution/public/detections/components/alerts_table/alerts_filter_group/index.tsx

```diff
@@ -6,14 +6,32 @@
  */
 
 import { EuiFilterButton, EuiFilterGroup } from '@elastic/eui';
+import { rgba } from 'polished';
 import React, { useCallback, useState } from 'react';
+import styled from 'styled-components';
 import { Status } from '../../../../../common/detection_engine/schemas/common/schemas';
 import * as i18n from '../translations';
 
 export const FILTER_OPEN: Status = 'open';
 export const FILTER_CLOSED: Status = 'closed';
 export const FILTER_IN_PROGRESS: Status = 'in-progress';
 
+const StatusFilterButton = styled(EuiFilterButton)<{ isActive: boolean }>`
+  ${(props) =>
```

Comment:
nit: I would spread props as you did in line 29.
```suggestion
  ${({ theme }) =>
```